### PR TITLE
feat: Create mktg card container gts

### PR DESCRIPTION
**Pull request by ChesneyJulian**
This component implements flexbox grid as a container for card components. Layout of cards can be determined by placing classes of col-12, col-3, etc on the cards themselves. It also has splattributes available for adding classes and other attributes to the container.

image

### CODE REVIEW

File: packages/ember/src/components/nrg/mktg/card-container.gts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  Blocks: {
+    default: [
+      {
+        Card: ComponentLike<Card>;
+      },
+    ];
+  };
+}
+const CardContainer: TOC<CardContainerSignature> = <template>
+  <div class="row justify-content-center p-2" ...attributes>
```
Comment (ChesneyJulian) [inline packages/ember/src/components/nrg/mktg/card-container.gts:17]: Change from flexbox grid to CSS grid

File: packages/ember-test-app/app/templates/cards.hbs [hunk 2]
```diff
@@ -1,5 +1,15 @@
-<div data-theme="marketing" class="container grid p-3">
-  <Nrg::Mktg::Card class="g-col-12 g-col-md-4" />
-  <Nrg::Mktg::Card class="g-col-12 g-col-md-4" />
-  <Nrg::Mktg::Card class="g-col-12 g-col-md-4" />
+<div data-theme="marketing" class="container">
+  <p class="py-3">Row on desktop, stacked on mobile</p>
+  {{!-- template-lint-disable no-inline-styles --}}
+  <Nrg::Mktg::CardContainer class="bg-info rounded" style="--bs-gap: .5rem;" as |CardContainer|>
```
Comment (TSenter) [inline packages/ember-test-app/app/templates/cards.hbs:4]: Is the default gap too high? If so, we can change it in `@nrg-ui/css`.

File: packages/ember-test-app/app/templates/cards.hbs [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+    <CardContainer.Card class="g-col-12 g-col-md-4" />
+    <CardContainer.Card class="g-col-12 g-col-md-4" />
+    <CardContainer.Card class="g-col-12 g-col-md-4" />
```
Comment (TSenter) [inline packages/ember-test-app/app/templates/cards.hbs:7]:
```suggestion
  <Nrg::Mktg::CardContainer class="bg-info rounded" style="--bs-gap: .5rem;" as |Container|>
    <Container.Card class="g-col-12 g-col-md-4" />
    <Container.Card class="g-col-12 g-col-md-4" />
    <Container.Card class="g-col-12 g-col-md-4" />
```

File: packages/ember-test-app/app/templates/cards.hbs [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+  </Nrg::Mktg::CardContainer>
+  <p class="py-3">stacked on mobile and desktop</p>
+  <Nrg::Mktg::CardContainer class="bg-secondary" as |CardContainer|>
+    <CardContainer.Card class="g-col-12" />
+    <CardContainer.Card class="g-col-12" />
+    <CardContainer.Card class="g-col-12" />
```
Comment (TSenter) [inline packages/ember-test-app/app/templates/cards.hbs:13]:
```suggestion
  <Nrg::Mktg::CardContainer class="bg-secondary" as |Container|>
    <Container.Card class="g-col-12" />
    <Container.Card class="g-col-12" />
    <Container.Card class="g-col-12" />
```

File: packages/ember-test-app/tests/integration/components/nrg/mktg/card-container-test.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+module('Integration | Component | nrg/mktg/card-container', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function () {
+    await render(hbs`
+      <Nrg::Mktg::CardContainer class="bg-info rounded" as |CardContainer|>
+        <CardContainer.Card class="first g-col-12 g-col-md-4" />
+        <CardContainer.Card class="second g-col-12 g-col-md-4" />
+        <CardContainer.Card class="third g-col-12 g-col-md-4" />
```
Comment (TSenter) [inline packages/ember-test-app/tests/integration/components/nrg/mktg/card-container-test.ts:14]:
```suggestion
      <Nrg::Mktg::CardContainer class="bg-info rounded" as |Container|>
        <Container.Card class="first g-col-12 g-col-md-4" />
        <Container.Card class="second g-col-12 g-col-md-4" />
        <Container.Card class="third g-col-12 g-col-md-4" />
```

**Review by TSenter — CHANGES_REQUESTED** [review]

File: packages/ember-test-app/app/templates/cards.hbs (hunk 2 above)
Comment (ChesneyJulian) [reply packages/ember-test-app/app/templates/cards.hbs]: I'm not sure I have an opinion on that, I was more just aiming to give examples of how you would go about changing that if needed. Bootstrap doesn't have a gap class available, so I wanted to highlight that it has to be done through css. We can discuss that further if you'd like

Comment (TSenter) [reply packages/ember-test-app/app/templates/cards.hbs]: In that case, I might argue that we should add some `gap` classes, even if all they do is set the CSS variable.

**Review by TSenter — APPROVED** [review]

File: packages/ember-test-app/app/templates/cards.hbs (hunk 2 above)
Comment (ChesneyJulian) [reply packages/ember-test-app/app/templates/cards.hbs]: Should that be done after Keith's design system PR? Or go ahead and do that now?

Comment (ChesneyJulian) [reply packages/ember-test-app/app/templates/cards.hbs]: I think the embroider error was due to the promo component being moved and not being updated in its implementation in the test app. Should be good now!

**Review by lnicely — DISMISSED** [review]
Looks good to me - let's have someone else review too before merging

**Review by TSenter — APPROVED** [review]

**Pull request merged**